### ISSUE TITLE
Data point 0 and 1, does not color correctly

### ISSUE THREAD

**Issue by Wiggyboy** [apexcharts/apexcharts.js#1742]
# Bug report
This relates to #286 3rd point in the list.
I understand it has been fixed? But I had this (or very similiar) issue today.

## Codepen
https://codepen.io/wiggyboy/pen/GRoywEp

## Explanation

- What is the behavior you expect?
I have configured custom `fillColor` and `strokeColor` for point 0 and 1. The way I configured I expect point 0 to have white fill, and black border (`strokeColor`). I expect point 1 to have black fill and gray border (`strokeColor`).
- What is happening instead?
The black fill and gray border (`strokeColor`) is shown on both. If I remove the configuration for point 0, it makes no difference. The configuration of point 1 is still drawn on both point 0 and 1. If I have no (standard configuration) on point 1, nothing special will be drawn on point 0 regardless of configuration.
Basically, somehow the points 0 and 1 is tied where configuration for 0 has no effect, and configuration on point 1 has the effect on both point 0 and 1. All other points works as expected.

**Comment by junedchhipa** [comment]
Thanks for reporting, fixed it!
